deps/jerry/jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -225,9 +226,7 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_FREEZE, "freeze")
 #if !defined (CONFIG_DISABLE_DATE_BUILTIN)
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_GET_DAY_UL, "getDay")
 #endif
-#if !defined (CONFIG_DISABLE_REGEXP_BUILTIN)
```

Comment:
This seems unrelated JerryScript problems?

Reply:
According to comment, lit-magic-strings.inc.h should be generated by gen-magic-strings.py from lit-magic-strings.ini, not by hand.
Seems upstream is not strictly following this rule.

Reply:
According to comment, lit-magic-strings.inc.h should be generated by gen-magic-strings.py from lit-magic-strings.ini, not by hand.
Seems upstream is not strictly following this rule.

Reply:
Okay, just created https://github.com/Rokid/ShadowNode/issues/283 for tracking this one, skip it now :p